charts/argo-workflows/values.yaml

```diff
@@ -576,6 +576,38 @@ server:
       #   hosts:
       #     - argoworkflows.example.com
 
+  GKEbackendConfig:
+    # -- Enable BackendConfig custom resource for Google Kubernetes Engine
+    enabled: false
+    # -- [BackendConfigSpec]
+    spec: {}
+  #  spec:
+  #    iap:
+  #      enabled: true
+  #      oauthclientCredentials:
+  #        secretName: argoworkflows-secret
+
+  ## Create a Google Managed Certificate for use with the GKE Ingress Controller
```

Comment:
I think this needs to be `# --` for helm-docs to pick it up correctly. Same a few lines down

Reply:
made the changes, thanks !

Reply:
I removed them again in [38c9d4e](https://github.com/argoproj/argo-helm/pull/2066/commits/38c9d4e248ab03d4c682844357dc9f40944aff07). We only want this comment on the "leaf" keys, otherwise they are added twice to the README

```
| server.GKEbackendConfig | object | `{"enabled":false,"spec":{}}` | Create a Google Backendconfig  for use with the GKE Ingress Controller |
| server.GKEbackendConfig.enabled | bool | `false` | Enable BackendConfig custom resource for Google Kubernetes Engine |
| server.GKEbackendConfig.spec | object | `{}` | [BackendConfigSpec] |
```